include/seahorn/Analysis/SeaBuiltinsInfo.hh

```diff
@@ -17,6 +17,7 @@ enum class SeaBuiltinsOp {
   ASSERT,             /* verifier.assert */
   ASSERT_NOT,         /* verifier.assert.not */
   IS_DEREFERENCEABLE, /* sea.is_dereferenceable */
+  ASSERT_IF,          /* sea.assert.if */
```

Comment:
would be useful to also add `sea.assert.not.if`. I use the `not` version of the functions to avoid creating extra negation instruction in llvm

Reply:
For now i am adding a verifier.assert.not and mapping to sea.assert.if... we can add it i run into problems with the extra negation instruction.